sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 11.2.0-beta.3 (Unreleased)
+## 11.2.0 (2021-02-10)
```

Comment:
We should add a note about bug fixes or including all the features in the various 11.2.0-beta releases.  We try not to have empty release sections.

Reply:
Sorry, I realized the layout of this file after pushing this out. I have now added some notes by looking at the commit history.

Reply:
As I ponder a near-future GA release for Key Vault, I'm actually pondering doing a roll-up of changes that made it to the GA so people don't have to aggregate themselves, especially if previous betas made a lot of renames. Customers probably just want to know that, for example,

```
* Added `SearchIndexingBufferedSender` to optimally batch upload document changes to an index.
```

Instead of...

```
## 11.2.0

### Breaking changes

* Renamed `FooBar` to `SearchIndexingBufferedSender`.

## 11.2.0-beta.2

### Breaking changes

* Renamed `TheOgClient` to `FooBar`

etc...
```

Thoughts? /cc @AlexGhiondea @annelo-msft @chidozieononiwu

Reply:
My 2c - one tenet I want to adhere to as much as possible (beyond fixing typos and incorrect messages) is to not touch the change log entries for earlier releases. This  gives a true picture of the changes and a clear view of the code changes both to developers and consumers.

Rolling up the changes is useful. However, does that bring the expectation that all changes will be represented in the GA notes? Say a feature was added in Preview 1. Will it be a mistake to not note it again in GA? If yes, that is pretty strict requirement and burden at GA time. If no, then the roll-up is mostly for noting useful things and cannot be relied upon to collect all changes.

Reply:
It may be a burden for us at GA, but 1) that doesn't happen nearly as often, and 2) it's better for our customers, is it not? Most are going to be interested in what changed since the last GA. Why make it harder to know what's changed?

Reply:
I'm also not advocating (as mentioned internally in the thread I started in our Teams channel) we remove or change the beta sections. They should stay. I just think we could do a better job highlighting the changes since our last GA. Considering we try to ship a sort of "RC" before each GA, that pretty much means that every GA will just be, "Please read everything preceding this and after the last GA, and figure out the final changes through many renames, etc."